docs/handbook/business-model.md

```diff
@@ -0,0 +1,39 @@
+# Business Model
+
+PostHog is a for profit company that balances the need to improve the open source code of PostHog with the need to add source-available features in order to generate income. We will build an open core business model.
+
+## Why would you work on the Community Edition?
+
+A concern could be that given our business model, we'd only work on paid features.
+
+The reality is that paid features can increase our revenue, thus our ability to grow and hire more developers, who we will use on both versions of the product. When we work on the Community Edition, it increases the community size, which means we end up with more features, and thus a better product. This means we get yet more community growth and it also helps with revenue growth too since the source-available product will also improve.
+
+At the moment, 100% of our focus is on the Community Edition of the software.
+
+## Promises
+
+1. We won't introduce features into the open source codebase with a delay.
+1. We will always release and open source all tests we have for an open source feature.
+1. The open source codebase will never contain arbitrary limits (i.e. event volumes, user numbers).
+1. The majority of new features made by PostHog will remain open source.
+1. The product will always be available for download without leaving an email address or logging in.
+1. We will always allow you to [benchmark](https://news.ycombinator.com/item?id=18103162) PostHog.
+
+## What features are paid only?
+
+If the wider community contributes a new feature, that isn't already a source-available feature, we aim nearly always for it to go into the open source codebase.
+
+When PostHog makes a new feature, we ask ourselves two questions:
+
+1. Who is the likely type buyer of this feature?
+1. Would this feature help more users find and use PostHog?
+
+If the likely buyer is an individual contributor, the feature will be open source. Otherwise, if the likely buyer is a manager, director or executive, it will be source available. The exception to this is if the feature will significantly help the community to increase. For example, initially we planned "multiple users" as a feature for the source-available version. However, we decided that having multiple users would help the community to grow, which benefits everyone disproportionately.
+
+## How open source benefits from open core.
+
+1. PostHog contributes many new features to the open source version. Having a viable business model makes it easier for us to invest more here.
+1. Security fixes.
+1. Support until the community can self sustain itself.
```

Comment:
Outside of grammatical ambiguity, has this ever happened/is this even a goal we should have?